cli/studio.py

```diff
@@ -86,9 +86,11 @@ def _build_single_scenario(clean: bool, allow_offset_map: bool, scenario: str):
 def _build_single_scenario_proc(
     clean: bool, allow_offset_map: bool, scenario: str, semaphore: Semaphore
 ):
-    semaphore.acquire()
-    _build_single_scenario(clean, allow_offset_map, scenario)
-    semaphore.release()
+    semaphore.lock()
```

Comment:
oops, sorry, I first wrote `lock()` by accident and then edited it to `acquire()`.  But you were too fast!  my bad! :)

Reply:
fixed in the latest commit:) Thanks for these suggestions Steve!